src/pages/news/news-add/news-add.js

```diff
@@ -149,46 +166,75 @@ const NewsAdd = () => {
       ))
       : null;
 
+  const handleAuthorImageLoad = (e) => {
+    if (e.target.files && e.target.files[0]) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        setFieldValue('authorPhoto', event.target.result);
+        setAuthorPhoto(event.target.result);
+      };
+      reader.readAsDataURL(e.target.files[0]);
+    }
+  };
+
+  const handleNewsImageLoad = (e) => {
+    if (e.target.files && e.target.files[0]) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        setFieldValue('newsImage', event.target.result);
+        setNewsImage(event.target.result);
+      };
+      reader.readAsDataURL(e.target.files[0]);
+    }
+  };
+
```

Comment:
you are already using the same code in news-form.js. maybe it makes sense to optimize it

Reply:
we can optimize the news page in the next sprint using the same form